tools/scripts-config-react-webpack/config/webpack.config.js

```diff
@@ -379,6 +381,17 @@ module.exports = ({ getUserConfig, mode }) => {
 						filename: getFileNameForExtension('css', cssPrefix),
 						chunkFilename: getFileNameForExtension('css', cssPrefix),
 					}),
+				isEnvProduction &&
+					process.env.SENTRY_AUTH_TOKEN &&
+					new SentryWebpackPlugin({
+						// see https://docs.sentry.io/product/cli/configuration/ for details
+						org: sentryConfig.org || process.env.SENTRY_ORG || 'talend-0u',
+						project: sentryConfig.project || process.env.SENTRY_PROJECT,
+						release: VERSIONS.version,
+						include: sentryConfig.include || ['dist/'],
+						ignore: sentryConfig.ignore || ['cdn/'],
```

Comment:
I wonder about path resolution, I expect the need for a fix at some point, but let's try as it is or verify by reading the the webpack plugin code to know how this is handled.
Remember this webpack file is in node_modules far from the src so it depends how the plugin resolve the path, if it use cwd or sth else.

Reply:
Tried to verify by reading the the webpack plugin code, but get lost. 😅 
Yes let's try then.
Not sure it counts (because monorepo), but tested on playground it works.